source/2020-05-29-the-ember-times-issue-150.md

```diff
@@ -9,11 +9,22 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Ember tutorials on MDN 🎉,
+"My Experience with Ember.js" Part 2 🎬,
+...
 
 READMORE
 
 ---
 
+## [Ember tutorials on MDN 🎉](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks#Ember_tutorials)
+
+[Preston Sego (@NullVoxPopuli)](https://github.com/nullvoxpopuli) shared the [news](https://twitter.com/nullvoxpopuli/status/1262124413794881541) that [Ember tutorials](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks#Ember_tutorials) are now on the Mozilla Developer Network (MDN) web docs, along with React and Vue! It's awesome to see Ember included as [MDN](https://developer.mozilla.org/) is one of the most respected resources on the web for frontend developers. 
```

Comment:
In the last sentence, although "as" works, I think the sentence may sound better as follows:
```suggestion
[Preston Sego (@NullVoxPopuli)](https://github.com/nullvoxpopuli) shared the [news](https://twitter.com/nullvoxpopuli/status/1262124413794881541) that [Ember tutorials](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks#Ember_tutorials) are now on the Mozilla Developer Network (MDN) web docs, along with React and Vue! It's awesome to see Ember included in [MDN](https://developer.mozilla.org/), one of the most respected resources on the web for frontend developers. 
```